boa3_test/tests/compiler_tests/test_range.py

```diff
@@ -204,32 +182,22 @@ def test_range_given_step(self):
             + Opcode.RET        # return
         )
 
-        path = self.get_contract_path('RangeGivenStep.py')
-        output = self.compile(path)
+        output, _ = self.assertCompile('RangeGivenStep.py')
         self.assertEqual(expected_output, output)
 
-        path, _ = self.get_deploy_file_paths(path)
-        runner = BoaTestRunner(runner_id=self.method_name())
+    async def test_range_given_step(self):
+        await self.set_up_contract('RangeGivenStep.py')
 
-        invokes = []
-        expected_results = []
-
-        invokes.append(runner.call_contract(path, 'range_example', 2, 10, 3))
-        expected_results.append([2, 5, 8])
-        invokes.append(runner.call_contract(path, 'range_example', -2, 10, 3))
-        expected_results.append([-2, 1, 4, 7])
-
-        runner.execute()
-        self.assertEqual(VMState.HALT, runner.vm_state, msg=runner.error)
-
-        for x in range(len(invokes)):
-            self.assertEqual(expected_results[x], invokes[x].result)
+        result, _ = await self.call('range_example', [2, 10, 3], return_type=list)
+        self.assertEqual([2, 5, 8], result)
+        result, _ = await self.call('range_example', [-2, 10, 3], return_type=list)
+        self.assertEqual([-2, 1, 4, 7], result)
```

Comment:
I know this is how the tests were before, but since we are refactoring this test case anyway, could you change the expected results to `list(range(args))`?
```suggestion
        result, _ = await self.call('range_example', [2, 10, 3], return_type=list)
        self.assertEqual(list(range(2, 10, 3)), result)
        result, _ = await self.call('range_example', [-2, 10, 3], return_type=list)
        self.assertEqual(list(range(-2, 10, 3)), result)
```